clang/test/SemaSYCL/unsupported_math.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clangxx --sycl %s -c -o a_kernel.bc -Xclang -verify
+
+#include <CL/sycl.hpp>
+
+int run_test_a(int v, cl::sycl::queue &deviceQueue) {
+    try {
+      int arr[] = {v};
+      {
+        cl::sycl::buffer<int, 1> buf(arr, 1);
+        deviceQueue.submit([&](cl::sycl::handler &cgh) {
+          cl::sycl::stream stream(1024, 80, cgh);
+          auto acc = buf.get_access<cl::sycl::access::mode::read_write>(cgh);
+          cgh.single_task<class kernel_a>([=]() {
+            acc[0] *= 2;
+            acc[0] += (int)__builtin_fabsf(-1.0f); // expected-error{{builtin is not supported on this target}}
+          });
+        });
+        deviceQueue.wait_and_throw();
+      }
+      return arr[0];
+    }
+    catch (cl::sycl::exception &e) {
```

Comment:
Please apply clang-format to the whole commit

Reply:
Done. Thanks